applications/nrf_desktop/doc/hid_forward.rst

```diff
@@ -10,6 +10,7 @@ HID forward module
 Use the |hid_forward| to:
 
 * Receive the HID input reports from the peripherals connected over Bluetooth®.
+* Can process the HID input reports in report or boot protocol.
```

Comment:
You have a grammar issue here:
"Use the |hid_forward| to: * Can process the HID input reports in report or boot protocol."
I guess this would do?
```suggestion
* Process the HID input reports in report or boot protocol.
```